### ISSUE TITLE
[BUG] Accessibility: The toggle to enable/disable scripts doesn't work properly with screen reader on Chrome.

### ISSUE THREAD

**Issue by chigkim** [Tampermonkey/tampermonkey#2676]
## Actual Behavior

When I toggle the Enable switch on the dashboard, or the toggle that appears after opening Tampermonkey from the toolbar, the label does not update. As a result, screen reader users cannot tell whether a script is enabled or disabled.

This works correctly in Firefox: the label updates between Enabled and Disabled.

## Expected Behavior

After toggling, the screen reader label should update to reflect the current state (Enabled or Disabled).

## Steps to Reproduce

1. Install the NVDA screen reader: https://www.nvaccess.org/download/
2. Turn NVDA on.
3. Navigate to a webpage where an active script is running.
4. Open Tampermonkey from the browser toolbar.
5. Use the Up/Down arrows to find the Enabled toggle with NVDA.
6. Press Space or Enter to toggle it.
7. Use the Up/Down arrows to return to the same toggle.
8. Observe that NVDA continues to announce Enabled, even when the script is disabled.

## Environment (please complete the following information)

- Browser and Version: Chrome Version 144.0.7559.97
- Tampermonkey version: v5.4.1
- Operating System and Version: Windows 10 pro 22H2 build 19045.6809

Thanks!

**Comment by derjanb** [comment]
5.5.6235 ([crx](https://tampermonkey.net/crx/tampermonkey_beta_current.crx)|xpi in review) now shows the current state on hovering using the title attribute. Does this appear on screen readers?

Chrome/Edge users, please update or install Tampermonkey BETA.
Firefox users, please wait for the xpi being reviewed.

For a quick fix please export your settings and scripts as zip or (JSON) file at the "Utilities" tab and import it back at the fixed BETA version.

**Comment by chigkim** [comment]
Unfortunately no.

I'm not sure if this is any helpful, but this is where the screen reader reads "Enabled" from inside the `<span>`.

`<table id="table_YWN0aW9udGFibGUtZW5hYmxlZF91bmRlZmluZWQ" class="actiontable at_enabled"><tr id="tr_RW5hYmxlZF9lbmFibGVk_outer" class="clickable"><td id="td_YWN0aW9udGRfRW5hYmxlZA_enabled"><i id="i_RW5hYmxlZF91bmRlZmluZWQ" class="far fa-times red"></i></td><td id="td_YWN0aW9udGRfRW5hYmxlZA_enabled" colspan="2"><span id="span_RW5hYmxlZF9lbmFibGVk_bu" class="greyed">Enabled</span></td></tr></table>`

**Comment by derjanb** [comment]
You're not on version 5.5.6235

**Comment by chigkim** [comment]
Hmm, I deleted the regular tampermonkey, unzip the crx that posted here, installed on Chrom with load unpacked. It said I have Tampermonkey BETA.
The version said 5.5.6234.

**Comment by chigkim** [comment]
Also the manifest.json inside tampermonkey_beta.crx says `"version": "5.5.6234",`
